src/datanode/src/datanode.rs

```diff
@@ -148,20 +148,26 @@ impl From<&DatanodeOptions> for StorageEngineConfig {
 #[serde(default)]
 pub struct ProcedureConfig {
     /// Storage config for procedure manager.
-    pub store: ObjectStoreConfig,
+    pub store_path: String,
+    pub max_retry_times: usize,
```

Comment:
suggestion: Please document each field in this config. What's the meaning of retry_delay/max_retry_times?
```suggestion
    /// .....
    pub max_retry_times: usize,
```